README.md

```diff
@@ -60,9 +62,9 @@ Daniel.Nt has played 3749 matches
   * [getStatus](#getStatus)
   * [Custom](#Custom)
 ---
-### [Changelog](#Changelog)
+### [Changelog](#changelog-1)
```

Comment:
When clicking on the current links they don't work in GitHub (they only in npm because of the anchor). The `#changelog` link won't work because, as you can see from both npm & GitHub, that links to the place where the reminders are. If you go to see the anchor for the actual sections you see they are `#changelog-1` & `#typescript-integrations-1`: you can try it by putting that in the URL even now.
Now that you made me think of it, old anchors are no longer needed and can be removed.

Reply:
oh nvm, im dumb, got it now Xd